resources/scripts/frontend/spritesvg.js

```diff
@@ -0,0 +1,3 @@
+const icons = require.context('@images/sprite-svg', true, /\.svg$/);
```

Comment:
This seems like it would require the user to restart their build process when they add a new svg file to the sprite directory for it to be detected.
A watcher plugin (e.g. something based on `chokidar`) should be added to re-trigger a build on file add/delete/change/etc. events. It would even be better if we can keep this logic out of the frontend bundle and keep it on the webpack side entirely.

Reply:
Hi @atanas-dev I've tested here and when add/delete/chante/etc happens the webpack will update the `sprite.svg`.